f2/src/pdf/MoneyLostInfoView.js

```diff
@@ -0,0 +1,74 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { containsData } from '../utils/containsData'
+import { testdata } from '../ConfirmationSummary'
+import { Text, View, Image } from '@react-pdf/renderer'
+import { pdfStyles } from './pdfStyles'
+import { DescriptionItemView } from './DescriptionItemView'
+import { formatList } from '../utils/formatList'
+import { formatDate } from '../utils/formatDate'
+import line from '../images/line.png'
+
+export const MoneyLostInfoView = (props) => {
+  const lang = props.lang
+
+  const methodPaymentSummary = []
+  let methodPaymentLine
+
+  const moneyLost = {
+    ...testdata.formData.moneyLost,
```

Comment:
Should we be importing `testdata` from the ConfirmationSummary component? If the goal is to use default values why not just import `defaultValues `like we do in the forms.

Reply:
Hi Justin, 
This PR is not ready for code review yet. I created this PR for business to try things out. I'll send you Code review when it is ready. Thanks!